protocol-units/bridge/service/src/chains/ethereum/client.rs

```diff
@@ -274,6 +276,7 @@ impl crate::chains::bridge_contracts::BridgeContract<EthAddress> for EthClient {
 		&mut self,
 		bridge_transfer_id: BridgeTransferId,
 		pre_image: HashLockPreImage,
+		framework: bool 
```

Comment:
I know this is still in draft, but I don't think we need framework bool arg here, we don't want to update the traits. When I suggested framework: bool arg, I was thinking of using that in the testing setup code to conditionally call the .move script, was just a loose idea. Feel free to disregard, looks like that arg is not being used anyway in the implementation of the trait methods here.